addons/test_mail_full/tests/test_ir_mail_server.py

```diff
@@ -0,0 +1,129 @@
+from contextlib import contextmanager
```

Comment:
as we depend on it, can you add a test to check that we can not write on `create_uid` ?